### DISCUSSION TITLE
ChartJS v3 Compatibility

### DISCUSSION THREAD

**Discussion by jithinvmohan** [chartjs/chartjs-plugin-deferred#26] [General]
Have been following the PRs for ChartJS v3 compatibility of this plugin. Do you know when will you be able to release a version compatible with v3?

Also would it not be better to keep the major version number aligned with ChartJS?

**Comment by simonbrunel** [comment]
@jithinvmohan [Version 2.0.0-beta.1](https://github.com/chartjs/chartjs-plugin-deferred/releases/tag/v2.0.0-beta.1) has just been released and is compatible with Chart.js v3 🎉 

> Also would it not be better to keep the major version number aligned with ChartJS?

That's a good question. I don't think it really makes sense to follow the major version of Chart.js because it's used to denote when there is a breaking change **in Chart.js**, not in plugins. So if we need to introduce a breaking change in plugins, we don't want to wait until Chart.js move to a new major version (which is not often) before we can release a major version of the plugins.

**Comment by LeeLenaleee** [comment]
@jithinvmohan V2 of the deferred plugin has been released as stable so it can be used with chart.js V3.
https://github.com/chartjs/chartjs-plugin-deferred/releases/tag/v2.0.0
